movo_common/movo_ros/src/movo/README.md

```diff
@@ -0,0 +1,118 @@
+<h1>Movo Xbox teleoperation</h1>
+
+To use the Xbox 360 wireless controller instead of the logitech controller, install the Xbox driver:
+
+`sudo apt-get install xboxdrv`
+
+You should plug a wireless controller receiver in Movo or your computer.
```

Comment:
Maybe add the development computer has to be in the same ROS network

Reply:
Done